djangocms_page_sitemap/sitemap.py

```diff
@@ -54,6 +54,15 @@ def items(self):
 
             if page_url.language in excluded:
                 continue
+            if is_versioning_enabled():
+                # render only published version of page contents.
+                page_content = PageContent.objects.filter(
```

Comment:
This is really bad for performance. I think we need to query better in the first place.